### PR TITLE
Main menu stuff

### PR DESCRIPTION
**Pull request by daniel112**
merging the new main menu functionality

### CODE REVIEW

**Comment by rraub** [comment]
waiting on fixing the pause issue

**Comment by daniel112** [comment]
pause menu is now fixed

File: src/AI.js [hunk 1]
```diff
@@ -115,6 +115,32 @@ function blockPath() {
 }
 
 function main() {
+  //Pause game stuff ////////////
+  $(document).bind("keydown", function(e){
+    console.log(e);
```
Comment (rraub) [inline src/AI.js]: you should remove this debugging statement before we merge so everyone doesn't have a much of logs in their console.

Comment (daniel112) [reply src/AI.js]: ok done

**Pull request merged**